### PR TITLE
Remove double ids

### PR DESCRIPTION
**Pull request by s-westphal**

### CODE REVIEW

**Comment by esc** [comment]
The ice of having no tests in `adapters.py` is getting dangerously thin.

**Comment by esc** [comment]
Maybe move the two functions to `fileio.py` and add some tests.

**Comment by s-westphal** [comment]
@esc I shifed the functions to utils.py and added some tests ;)

File: src/main/python/gulpio/fileio.py [hunk 1]
```diff
@@ -453,7 +451,7 @@ def __init__(self, adapter, output_folder, videos_per_chunk, num_workers):
         self.num_workers = int(num_workers)
 
     def __call__(self):
-        ensure_output_dir_exists(self.output_folder)
+        os.makedirs(self.output_folder, exist_ok=True)
```
Comment (esc) [inline src/main/python/gulpio/fileio.py]: Good call!

**Comment by esc** [comment]
Great, thanks!

**Pull request merged**